substrate/frame/contracts/uapi/src/host.rs

```diff
@@ -292,6 +292,17 @@ pub trait HostFn {
 	/// Returns the size of the pre-existing value at the specified key if any.
 	fn clear_storage_v1(key: &[u8]) -> Option<u32>;
 
+	/// Clear the value at the given key in the contract transient storage.
+	///
+	/// # Parameters
+	///
+	/// - `key`: The storage key.
+	///
+	/// # Return
+	///
+	/// Returns the size of the pre-existing value at the specified key if any.
+	fn clear_transient_storage(key: &[u8]) -> Option<u32>;
```

Comment:
Should those be marked as unstable here?

Reply:
Yes. I missed that. They should use the `deprecated` attribute to warn users about their instability.

Reply:
done